src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.cpp

```diff
@@ -113,6 +113,7 @@ LCAOrbitalBuilder::LCAOrbitalBuilder(ParticleSet& els, ParticleSet& ions, Commun
   std::string keyOpt("NMO");       // Numerical Molecular Orbital
   std::string transformOpt("yes"); // Numerical Molecular Orbital
   std::string cuspC("no");         // cusp correction
+  PosType twist(0.0);              // Supertwist coordinates
```

Comment:
It is better to use SuperTwist which reduces confusion. For the xml input, we have to live with the "twist" tag